src/app/parcels/parcelsTable/ParcelsPage.tsx

```diff
@@ -326,7 +400,11 @@ const ParcelsPage: React.FC = () => {
                             DbParcelRow,
                             string | DateRangeState | string[]
                         >
-                            dataPortion={parcelsDataPortion}
+                            dataPortion={
+                                isPackingManagerView
+                                    ? packingManagerViewDataPortion
+                                    : parcelsDataPortion
+                            }
```

Comment:
Is there a way we can avoid calculating both of these every time, instead just calculating the one we need?